docs/topics/api/addons.rst

```diff
@@ -44,6 +44,7 @@ This endpoint allows you to search through public add-ons.
     :query string appversion: Filter by application version compatibility. Pass the full version as a string, e.g. ``46.0``. Only valid when the ``app`` parameter is also present.
     :query string author: Filter by exact author username. Multiple author names can be specified, separated by comma(s), in which case add-ons with at least one matching author are returned.
     :query string category: Filter by :ref:`category slug <category-list>`. ``app`` and ``type`` parameters need to be set, otherwise this parameter is ignored.
+    :query boolean featured: Filter to only featured add-ons. The value `true` is optional - ``featured`` and ``featured=true`` are identical.
```

Comment:
This should be "Filter to load only featured add-ons." I think.

Reply:
It wasn't a spelling mistake (this time).  Tbh not sure about "load'.  I was attempting to match the other descriptions, but "Filter by featured (status)" seemed inaccurate because it's effectively just a boolean flag.